ioctl/cmd/contract/contracttestfunction.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package contract
+
+import (
+	"math/big"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-address/address"
+
+	"github.com/iotexproject/iotex-core/ioctl/cmd/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	testFunctionCmdUses = map[config.Language]string{
+		config.English: "function (CONTRACT_ADDRESS|ALIAS) ABI_PATH FUNCTION_NAME [AMOUNT_IOTX] " +
```

Comment:
how is this diff from contractinvokefunction.go?

Reply:
maybe i miss something, are you adding new cmds in this PR?
looks like it is same as existing contractinvokefunction.go

Reply:
It use Read() rather than Execute()